core/windows_and_tabs/window_snap.py

```diff
@@ -275,11 +287,13 @@ class Actions:
     def snap_window(pos: RelativeScreenPos) -> None:
```

Comment:
I know this isn't your code, but this is a tiny bit more pythonic

```suggestion
    def snap_window(position: RelativeScreenPos) -> None:
```